### DISCUSSION TITLE
Quality Difference in Spotify stream depending on Spotify / Sonos app and Music Assistant

### DISCUSSION THREAD

**Discussion by avidoni** [music-assistant/support#3390] [Q&A] [answered]
Hello

I have noticed a change in the sound quality of a spotify stream depending on wheter I use the native spotify app / webplayer / sonos app or whether i use the music assistant integration. The spotify/sonos integration seems to be more "clear" and crisp, in my ears it sounds better (i am in no way an audiophile, maybe its just EQd differently, but there seems to be more sound clarity) This happens whenever Musicassitant displays "external source: spotify".  I am aware of the -17dB renormalisation that music assitant does, but this is not a volume issue. I also tried to find whether spotify / sonos EQs some stuff on the way, but in both apps the equalizer was turned off. The Music Assitant Sound Quality is given as 44.1kHz/16bits oggvorbis.

Additionally to my question, do you guys know whether sonos trueplay also works when i use music assitant? As i dont have apple products, I currently cannot configure it anyways but it will be possible in the future.

**Comment by OzGav** [comment]
If the Trueplay  corrections are stored on the device, which I would expect they would, then yes it would work.

As for the current situation there is a new feature for each player that we haven’t had time to document yet. Go to the settings for the player and look for a button OPEN DSP SETTINGS and have a play with that.

**Comment by avidoni** [comment]
Thank you very much for the heads up on the new feature! as i understood it, this is a new advanced equalizer, which would apply the setting regardless of the played source. Im mainly interested in the cause of the different "qualities/eq settings", at which point the signal starts to differ etc. As there will be times where we still use spotify connect to initially start music, the additional EQ would not work in that case, right?

**Reply by OzGav** [reply]
It is difficult to say as no one else has reported this other than people who didn’t like the normalisation feature. We also don’t know if MA is passing the signal unaltered and the other integration is modifying it and you happen to like that.

**Reply by OzGav** [reply]
There is info here which may be of interest https://music-assistant.io/faq/tech-info/

**Comment by marcelveldt** [comment, marked as answer]
There is no difference in audio quality and I also must say that its simply not possible, Sonos received the exact same ogg vorbis file/stream from Spotify as we do, unless they secretly get a lossless cd quality version but I HIGHLY doubt that.

I hope you did not enable the option to send the audio stream as MP3 ?
